core/lb.go

```diff
@@ -14,17 +14,21 @@ import (
 
 var ErrTranscoderBusy = errors.New("TranscoderBusy")
 var ErrTranscoderStopped = errors.New("TranscoderStopped")
+var DetectorProfile ffmpeg.DetectorProfile
```

Comment:
Can we add a comment here as a reminder that this is just for convenience for now since we only support pre-loading the model for a single detector profile?

Reply:
Will do!